benchmarks/templates/benchmarks/tutorial_deepdive_1.html

```diff
@@ -99,19 +122,21 @@ <h3 class="benefits_heading is-size-3-mobile">Step 4: Modify the  __init__.py  f
                            ecosystem at large. You are registering your model to a unique global identifier. You can
                            see that this file is fairly straightforward. Lines 1-3 are standard imports, and line 5
                            adds the plugin to the registry. Let’s go ahead and modify the identifier of our model,
-                           from resnet-50 to resnet-50_youremail_sample. We can do this by changing line 5 to this:
+                           from resnet-50 to resnet50_tutorial_your-email. We can do this by changing line 5 to this:
```

Comment:
maybe username instead of email? not sure `resnet50_tutorial_jsmith@hotmail.com` is what we want to encourage.

Could even do `<username>/<identifier>`, e.g. `jsmith/resnet50`. @kvfairchild thoughts from the plugin management perspective?